test/unit/lib/plugins/aws/package/lib/addExportNameForOutputs.test.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const chai = require('chai');
+const runServerless = require('../../../../../../utils/run-serverless');
+
+const expect = chai.expect;
+
+describe('test/unit/lib/plugins/aws/package/lib/addExportNameForOutputs.test.js', () => {
+  let outputs;
+
+  before(async () => {
+    const result = await runServerless({
+      fixture: 'aws',
+      command: 'package',
+      configExt: {
+        service: 'myService',
+        resources: {
+          Outputs: {
+            ServerlessDeploymentBucketName: {
```

Comment:
I believe we shouldn't override it as a part of `resources` section as that output will be always generated